### ISSUE TITLE
feature request: Add exercises to each chapter

### ISSUE THREAD

**Issue by fmustafa17** [peterfriese/Asynchronous-Programming-with-SwiftUI-and-Combine#27]
Only Chapters 1, 2, 8, and 9 have exercises. The book would have a greater impact if there were exercises at the end of every chapter

**Comment by byte-lingual** [comment]
Agree. And a link to recommended blogs or forums for the book where learners can exchange help. Peter mentioned he used to post to a blog helping others what he learned when he first started using SwiftUI*pg.xvii. A referenced forum for the SwiftUI learners to help and support each other would be very useful.
